_scripts/collect_contributor_data.r

```diff
@@ -0,0 +1,33 @@
+
+if (!require(gh)) {
+    install.packages("gh")
+}
+if (!require(allcontributors)) {
+    install.packages("allcontributors")
+}
+
+library(allcontributors)
+library(gh)
+
+org_name <- "epiverse-trace"
+repos <- gh("GET /orgs/:org/repos", org = org_name)
```

Comment:
I didn't immediately catch how you excluded repos like `varnish` so probably good to make it explicit via a comment.

```suggestion
# We don't want to get contributions to forks by default to not count 
# contributions that happened upstream, outside of Epiverse
repos <- gh("GET /orgs/:org/repos", org = org_name)
```